plugins/inputs/fireboard/fireboard.go

```diff
@@ -110,7 +110,7 @@ func (r *Fireboard) Gather(acc telegraf.Accumulator) error {
 func scale(n int) string {
 	switch n {
 	case 1:
-		return "Celcius"
+		return "Celsius"
```

Comment:
We cannot do this unfortunately as this will break existing queries because the metric changes...